frontend/src/views/AmendingLawMetadataEditor.vue

```diff
@@ -206,7 +216,13 @@ const {
               v-for="element in elements"
               :key="element.eid"
               :to="{
-                name: 'AmendingLawMetadataEditorElement',
+                name:
```

Comment:
Suggestion: this would be a good candidate to ["pre-compute"](https://vuejs.org/guide/essentials/computed.html#computed-caching-vs-methods) i.e. something like

```ts
const elementLinks = computed(() => {
  return elements.value.map(el => {
    // return the el itself + the route it should link to
  })
})
``` 

... and then iterate over `elementLinks` instead.

Advantages: It's faster and the template is a bit more tidy